projects/rocprofiler-sdk/source/lib/output/domain_type.hpp

```diff
@@ -40,6 +40,8 @@ enum class domain_type
     ROCDECODE,
     ROCJPEG,
     PC_SAMPLING_STOCHASTIC,
+    STREAMING_PERFORMANCE_MONITOR,
+    SPM_COUNTER_VALUES,
```

Comment:
```suggestion
    STREAMING_PERFORMANCE_MONITOR,
    STREAMING_PERFORMANCE_MONITOR_VALUES,
```
Weird mix of fully spelling out SPM and then abbreviating it. Keep it consistent